files/default/neutron-ha-tool.py

```diff
@@ -104,14 +107,19 @@ def parse_args():
                          'is mainly useful when evacuating routers from '
                          'a node where the l3-agent is not running for '
                          'some reason.')
+    ap.add_argument('--agent-selection-mode', choices=['random', 'least-busy'],
+                    default='least-busy',
+                    help='Determines how target agent is selected for routers '
+                         '"random" selects target agent randomly, '
+                         '"least-busy" selects the least busy agent.')
```

Comment:
@vuntz Nope, the `argparse` module automatically does line-wrapping by default, although I admit that this is quite unclear from the docs:

https://docs.python.org/2/library/argparse.html#formatter-class